### ISSUE TITLE
Browser Font choices not respected when this addon is enabled

### ISSUE THREAD

**Issue by gggirlgeek** [asger-finding/resistfingerprinting-toggle#149]
When "Resist Fingerprinting Toggle 0.1.0" is enabled Waterfox and Firefox do not respect my font choices. When I disable the extension, then restart the browser, my font choices are under my control again, and pages render with the forced font I choose under Settings >> Fonts >> Default Font >> Advanced, and DISABLED "Allow sites to choose their own fonts...".  If I do not restart the browser after enabling or disabling this addon, I see no change from the current state of the fonts rendered. Also it does not matter whether I have toggled fingerprinting on or off. So it took me weeks to figure out what was causing the issue.

When I use another addon: "[Toggle Resist Fingerprinting 1.2](https://github.com/Aaron-P/ToggleResistFingerprinting)", my fonts obey my choices, whether fingerprinting is enabled or disabled.

I realize both of the addons I mentioned may be retired because Firefox and many other browsers have inbuilt options to resist fingerprinting now. However, I prefer a convenient toggle button on my toolbar so I can quickly eliminate sources of problems with websites without hunting in multiple places for settings. I have other toggle addons, too, that help with basic settings, so your addon fits in nicely.

Basically just reporting the bug. I'll use the other addon for now.

Thanks a lot!
```
Waterfox 6.0.5/ Firefox 119.0
Windows 11 Pro 23H2
Preferred fonts: "Arial Rounded MT", or "Outfit"
Note: I have the font added to my "Firefox\fonts\" folders to make sure they're available if I move computers.
```
